ivy_tests/test_ivy/test_frontends/test_tensorflow/test_linalg.py

```diff
@@ -45,3 +47,40 @@ def test_tensorflow_det(
         fn_tree="linalg.det",
         input=np.asarray(x, dtype=input_dtype),
     )
+
+
+# slogdet
+@handle_cmd_line_args
+@given(
+    dtype_x=helpers.dtype_and_values(
+        available_dtypes=ivy_np.valid_float_dtypes,
+        min_value=0,
+        max_value=50,
+        shape=helpers.ints(min_value=2, max_value=20).map(lambda x: tuple([x, x])),
```

Comment:
Please update `max_value` to have 10 instead, as it's the default value of `max_dim_size` in the `dtype_and_values` helper.

Reply:
done